src/test/ui/self/arbitrary_self_types_pin_lifetime_mismatch.base.stderr

```diff
@@ -13,7 +13,7 @@ LL |     fn a<'a>(self: Pin<&'a Foo>, f: &'a Foo) -> &Foo { f }
    |         ++++            ++           ++
 
 error[E0623]: lifetime mismatch
-  --> $DIR/arbitrary_self_types_pin_lifetime_mismatch.rs:8:76
```

Comment:
We lose the suggestion above. Can you file an issue for this?

Reply:
Filed https://github.com/rust-lang/rust/issues/96157